staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -262,8 +262,8 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 
 	availableController, err := statuscontrollers.NewAvailableConditionController(
 		informerFactory.Apiregistration().V1().APIServices(),
-		c.GenericConfig.SharedInformerFactory.Core().V1().Services(),
```

Comment:
Having `// HACK: ...` comments in the changed places in the code would be great.

Reply:
That's why we have `git` commits :)

Someone will simply revert this when they work on the real issue 194